styles.css

```diff
@@ -0,0 +1,138 @@
+
+body {
+    margin-left: 95px;
+    margin-right: 95px;
+    background-color: rgb(255, 250, 250);
+    max-width: 900px;
+    margin: auto auto;
+}
+
+h1 {
+    box-shadow: rgba(50, 50, 93, 0.25) 0px 30px 60px -12px inset, 
+    rgba(0, 0, 0, 0.3) 0px 18px 36px -18px inset;
+    text-align: center;
+    margin-top: 0;
+    margin-bottom: 10px;
+    font-size: 40px;
+    padding-top: 0;
+    padding-bottom: 0px;
+}
+
+.color, .font, .text {
+    display: block;
+    width: auto;
+    border:2px solid rgb(128, 128, 128);
```

Comment:
minor nit: sometimes space between selector and rule, sometimes not. There are formatters like [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) that can automate this